test/TestExpectations.json

```diff
@@ -2252,7 +2252,7 @@
   {
     "testIdPattern": "[launcher.spec] Launcher specs Puppeteer Puppeteer.launch should be able to launch Firefox",
     "platforms": ["darwin", "linux", "win32"],
-    "parameters": ["firefox", "headless"],
+    "parameters": ["chrome-headless-shell", "firefox"],
```

Comment:
Wrong? Params are `AND` logic. There are more instances of this change please revert. Also I think for FF we can drop the headless in params, wdyt?